R/dplyr-joins.R

```diff
@@ -520,6 +595,19 @@ inner_join <- function(
   dropreport   <- args_check$dropreport
 
   # Column names -----------------------------------
+  byexp <- grep(pattern = "==?",
```

Comment:
I think these lines of code should be converted to a function that can be called everywhere. Right now, the same code is repeated in multiple places.